packages/-ember-data/tests/unit/store/adapter-interop-test.js

```diff
@@ -40,9 +42,14 @@ module('unit/store/adapter-interop - DS.Store working with a DS.Adapter', functi
   testInDebug('Adapter can not be set as an instance', function(assert) {
     assert.expect(1);
 
-    store = DS.Store.create({
+    const BadStore = DS.Store.extend({
```

Comment:
lets keep the left side and see if it passes.

Reply:
@igorT I confirmed that using `Store.create` is not currently supported. The reason this test changed is we now do `getOwner(this)` in the constructor, but we also use getOwner in adapterFor and serializerFor.